html/elements/menu.json

```diff
@@ -288,8 +366,8 @@
                 }
               },
               "status": {
-                "experimental": true,
-                "standard_track": true,
+                "experimental": false,
+                "standard_track": false,
```

Comment:
Another candidate for deletion?

Reply:
Also part of https://github.com/mdn/browser-compat-data/pull/6955.